src/components/separator.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export default styled.div`
+    display: flex;
+    align-items: center;
+    
+    &:before, &:after {
+        content: '';
+        width: 100%;
+        height: ${props => props.height || 3}px;
+        background: ${props => props.color || '#000'};
+    }
```

Comment:
Sure you can use it.
Space around title should be implemented via its margin, I believe.